lib/services/github.ts

```diff
@@ -101,14 +115,16 @@ export class GithubService extends WorkerClient<string> implements ServiceListen
 		super();
 
 		// Determine type of service.
-		if (constObj.loginType.type !== 'integration') {
-			throw new Error('Do not yet support non-Integration type clients');
+		if (constObj.authentication.type === 'app') {
+			const appLogin = <GithubApp>constObj.authentication;
+
+			// Set the App ID
+			this.appId = appLogin.appId;
+			this.pem = appLogin.pem;
+		} else {
+			const userLogin = <GithubPAT>constObj.authentication;
```

Comment:
If you implement the `type: 'pat'` change in my comment above, you shouldn't need the cast here or in the previous block: TS should be able to know what type `authentication` has based on the `if` test above.